src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable max-len */
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PersonLink } from '../PersonLink/PersonLink';
+import { getPeople } from '../../api';
+
+const getPersonString = (name: string, born: number) => {
+  return `${name.toLocaleLowerCase().replaceAll(' ', '-')}-${born}`;
+};
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loader, setLoader] = useState<boolean>(true);
+  const [hasError, setHasError] = useState<boolean>(false);
+  const { personName } = useParams();
+
+  useEffect(() => {
+    getPeople()
+      .then(setPeople)
+      .catch(() => setHasError(true))
+      .finally(() => setLoader(false));
+  }, []);
+
+  return (
+    <div className="container">
+      <h1 className="title">People Page</h1>
+      <div className="block">
+        <div className="box table-container">
```

Comment:
I would also create a separate PoepleTable component and then put it in the PeoplePage